src/app.ts

```diff
@@ -1,13 +1,28 @@
 import express, { Application, Response } from "express";
+import {monitorService} from "./Monitoring/monitoring.js";
```

Comment:
`js?`

Reply:
added "--experimental-specifier-resolution=node" flag to the package.json. 
Please refer https://stackoverflow.com/a/65588027